v1alpha1/proto/grafeas.proto

```diff
@@ -16,1093 +16,1352 @@ syntax = "proto3";
 
 import "google/protobuf/empty.proto";
 import "google/api/annotations.proto";
+import "google/protobuf/field_mask.proto";
 import "google/protobuf/timestamp.proto";
 import "google/longrunning/operations.proto";
-import "google/devtools/source/v1/source_context.proto";
 
 package grafeas.v1alpha1.api;
 option go_package = "grafeas";
 
 // Request to get a Occurrence.
 message GetOccurrenceRequest {
-    // The name of the occurrence in the form
-    // "projects/{project_id}/occurrences/{occurrence_id}"
-    string name = 1;
+  // The name of the occurrence of the form
+  // "projects/{project_id}/occurrences/{OCCURRENCE_ID}"
+  string name = 1;
 }
 
 // Request to list occurrences.
 message ListOccurrencesRequest {
-    // This contains the projectId for example: projects/{project_id}
-    string parent = 5;
-    // The filter expression.
-    string filter = 2;
-    // Number of occurrences to return in the list.
-    int32 page_size = 3;
-    // Token to provide to skip to a particular spot in the list.
-    string page_token = 4;
+  // The name field contains the project Id. For example:
+  // "projects/{project_id}
+  // @Deprecated
+  string name = 1;
```

Comment:
I intentionally left fields we'd deprecated out of the oss proto. Please remove.

Reply:
Done.